python/alibiexplainer/pyproject.toml

```diff
@@ -12,7 +12,7 @@ packages = [
 [tool.poetry.dependencies]
 python = ">=3.8,<3.12"
 kserve = { path = "../kserve", extras = ["storage"], develop = true }
-alibi = { version = "^0.9.4", extras = ["shap", "tensorflow"] }
+alibi = { version = ">=0.9.4", extras = ["shap", "tensorflow"] } # From 0.9.5 alibi uses BSL license
```

Comment:
We cannot have alibi >= 0.9.5

Let's keep this unchanged here for now and I'll remove this as part of https://github.com/kserve/kserve/issues/3380

Reply:
My Bad. What I intended to do is <=. 😅

Reply:
Ha that makes sense

Reply:
left it as it is.